CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - [Feature] Adds `plot.PrecisionRecall` object-oriented API [#189]
 - [Feature] Adds `plot.pca`
 - [Doc] User guide on PCA
+- [Fix] 'ax' parameter removed from ROC constructor([#255](https://github.com/ploomber/sklearn-evaluation/issues/255))
```

Comment:
It seems to be failing on the 'check' CI check because it's a minor instead of major version?

Also, sorry that it changed another file— I ran `black .`, so I guess it automatically changed some formatting elsewhere.

Reply:
yes, since this is an API change, it should be a major version. we already upgraded the version so rebase from master, force push and the check should pass

Reply:
So should it go under 0.10.1dev or 0.10.0?

Reply:
0.10.0dev, but you shouldn't change it. I changed it already in the master branch, so rebase your branch and you'll get the right version